irohad/model/commands/create_role.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef IROHA_CREATE_ROLE_HPP
+#define IROHA_CREATE_ROLE_HPP
+
+#include "model/command.hpp"
+#include <string>
+
+namespace iroha {
+  namespace model {
+
+    /**
+     * Create new role in the system
+     */
+    struct CreateRole : public Command {
```

Comment:
All commands I think can be merged to one file

Reply:
Maybe, but outside of this pr